MovingBlocks/Terasology
# feat: Add AFK state for players in multiplayer

### Contains
As a gamer sometimes I have to go away from the keyboard for some reason and I want to tell my friends who I'm playing with that I'm away so they won't get mad at me or something. So, I made an `afk` command to do that goal.

### How to test
- Pull from [My Fork](https://github.com/iHDeveloper/Terasology) from branch `afk-command`
- Run the game from the source code
- Make sure you in multiplayer mode like (Hosting game or Playing online)
- Show the game console
- Type `afk`
- You can now go away without telling your friends that I'm AFK in the chat :smile: 

### Issues
> No more issues finally 😁 
- [x] Client can't send request to the **server**
- [x] Rebuild the way that client and server communicate in the `AFKSystem`
- [x] Auto disable the AFK when the player moves
- [x] Test it
